src/objects/token/models.py

```diff
@@ -33,6 +33,18 @@ class TokenAuth(models.Model):
     created = models.DateTimeField(
         _("created"), auto_now_add=True, help_text=_("Date when the token was created")
     )
+    application = models.CharField(
+        _("application"),
+        max_length=200,
+        blank=True,
+        help_text=_("Application which has an access to the API"),
```

Comment:
> which has an access

The "an" should be left in all 3 help_texts.

Reply:
done